charts/elasticsearch/values.yaml

```diff
@@ -207,3 +207,8 @@ nginx:
   fullnameOverride: ~
   ingressClass: ~
   resources: {}
+
+# sysctl init container
```

Comment:
We should put a warning here that sysctl affects the entire node, so all containers in all pods on the node will be affected by the sysctl changes.